habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -111,13 +117,12 @@ def get_observation(
 @registry.register_sensor
 class HabitatSimDepthSensor(DepthSensor):
     sim_sensor_type: habitat_sim.SensorType
-    sim_sensor_subtype: habitat_sim.SensorSubType
+    _sensor_spec_factory = habitat_sim.CameraSensorSpec
```

Comment:
Is `_sensor_spec_factory` accurate? Naming factory is used for Factory Class instance that can produce children of the abstract class. Here `_sensor_spec` may be better name?

Reply:
I see, is that kind of a function that generates a default SensorSpec/Config based on `sim_sensor_cfg = sensor._sensor_spec_factory()`? In that case, maybe something along `default_spec_generator` lines will be cleaner as the Factory has different meaning in OOP world.

Reply:
Generator is also an overloaded term in Python and it's not a generator.